common/static/coffee/src/discussion/views/discussion_thread_view.coffee

```diff
@@ -278,7 +278,6 @@ if Backbone?
       comment = new Comment(body: body, created_at: (new Date()).toISOString(), username: window.user.get("username"), votes: { up_count: 0 }, abuse_flaggers:[], endorsed: false, user_id: window.user.get("id"))
       comment.set('thread', @model.get('thread'))
       @renderResponseToList(comment, ".js-response-list")
-      @renderAttrs()
```

Comment:
This is what was causing the bug. In this context, it would `renderAttrs` on the thread object, which would lead to this section of code: https://github.com/edx/edx-platform/blob/master/common/static/coffee/src/discussion/views/discussion_content_view.coffee#L19

If the thread was not editable or deletable by the current user, it would disable these values with `@$(".action-edit").closest(".actions-item").addClass("is-hidden")`. However, the selector `@$(".action-edit").closest(".actions-item")` was selecting *all* items with class `actions-item` on the page, effectively hiding all of them.

Then this ajax callback (https://github.com/edx/edx-platform/blob/master/common/static/coffee/src/discussion/views/discussion_thread_view.coffee#L292) would resolve, and the line `comment.updateInfo(data.annotated_content_info)` would update the newest comment to the correct values.

So that's how we ended up with the main thread item and the most recent comment having correct buttons appearing, but everything else being wrong. This call does not need to be made at all, as the new comment gets it's attrs from the ajax callback just fine. Simply removing it is the cleanest fix.